src/index.html

```diff
@@ -5,8 +5,409 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a class="logo__link" href="#">
+          <img src="images/logo.png" alt="Logo Moyo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="table">
+      <div class="card table__element" data-qa="card">
+        <img
+          class="card__photo"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&quot Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <h3 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__mark">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__cost">
+          <p class="card__word-price">
+            Price:
+          </p>
+          <p class="card__price">
+            $2,199
+          </p>
+        </div>
+
+        <div class="button-buy">
+          <a
+            data-qa="card-hover"
+            class="button-buy__link"
```

Comment:
```suggestion
          <a
            data-qa="card-hover"
            class="card__button"
```
There is no need to make button a separate BEM block. It's just an element of card.
Also, you can just style the link correctly, no need to wrap it into div.